ApplicationLayer/Revisal/Revisal_Open_Application.xaml

```diff
@@ -0,0 +1,85 @@
+<Activity mc:Ignorable="sap sap2010" x:Class="Revisal_Open_Application" xmlns="http://schemas.microsoft.com/netfx/2009/xaml/activities" xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" xmlns:mva="clr-namespace:Microsoft.VisualBasic.Activities;assembly=System.Activities" xmlns:sap="http://schemas.microsoft.com/netfx/2009/xaml/activities/presentation" xmlns:sap2010="http://schemas.microsoft.com/netfx/2010/xaml/activities/presentation" xmlns:scg="clr-namespace:System.Collections.Generic;assembly=mscorlib" xmlns:sco="clr-namespace:System.Collections.ObjectModel;assembly=mscorlib" xmlns:ui="http://schemas.uipath.com/workflow/activities" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">

```

Comment:
Change the selector for the site, the title as is is not very stable. Check the other places where you used the same selector and change them there as well

Reply:
Changed selector and created out argument of type Window and passed it to (almost) every workflow that uses the revisal app.